conduct.md

```diff
@@ -132,7 +132,7 @@ TODO: Write an introductory paragraph.
 
 ### Infractions
 
-TODO: Write an introductory paragraph.
+Infractions are the punishments handed out to those who break the rules or don't follow class conduct one or more times. The more that the class conduct is broken by the individual, the more severe the penalty is. Infractions are split upon different categories and each infraction category adds to a point total when committed. For example, if you commit three Level 1 infractions, that is just the same as committing 1 Level 2 infraction. The higher the total infraction points you have, the worse the punishment is.
```

Comment:
It seems there might be a misunderstanding of the term "infraction" here. Infractions are the actual violations.

Reply:
Additionally, is "class conduct" the way that the class has decided to refer to the conduct guide within the conduct guide? The class should agree on one way of referring to the conduct guide.